Lib/multiprocessing/managers.py

```diff
@@ -1159,16 +1159,21 @@ def __imul__(self, value):
         self._callmethod('__imul__', (value,))
         return self
 
+    __class_getitem__ = classmethod(types.GenericAlias)
+
 
-DictProxy = MakeProxyType('DictProxy', (
+BaseDictProxy = MakeProxyType('DictProxy', (
     '__contains__', '__delitem__', '__getitem__', '__iter__', '__len__',
     '__setitem__', 'clear', 'copy', 'get', 'items',
-    'keys', 'pop', 'popitem', 'setdefault', 'update', 'values'
+    'keys', 'pop', 'popitem', 'setdefault', 'update', 'values',
```

Comment:
nit: don't think there's any reason to make a change to this line

```suggestion
    'keys', 'pop', 'popitem', 'setdefault', 'update', 'values'
```